rero_ils/modules/entities/templates/rero_ils/_local_organisation.html

```diff
@@ -0,0 +1,28 @@
+{# -*- coding: utf-8 -*-
+
+  RERO ILS
+  Copyright (C) 2019-2023 RERO
+  Copyright (C) 2019-2023 UCLouvain
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#}
+{{ dl(_('Name'), record.name) }}
```

Comment:
Is this will create a `dl + dt + dd` for each field ? A little bit overkill for HTML DOM

Reply:
The `dl` macro actually only generates `dt`+ `dd`. The name of the macro was poorly chosen.

Reply:
Ok, but for each field you will create a `DL` tag... This is not the best thing to do into HTML DOM. In a better world we only need one `DL` tag with multiple `dt+dd` tags into.